src/pages/guidelines/color/implementation.mdx

```diff
@@ -0,0 +1,546 @@
+
+<Row>
+<Column colLg={8}>
+
+<Tabs>
+
+<Tab label="White">
+
+![Layer set tokens applied to dropdown in the White theme](images/color-implementation-contextual-W.png)
+
+</Tab>
+
+<Tab label="Gray 10">
+
+![Layer set tokens applied to dropdown in the Gray 10 theme](images/color-implementation-contextual-G10.png)
+
+</Tab>
+
+<Tab label="Gray 90">
+
+![Layer set tokens applied to dropdown in the Gray 90 theme](images/color-implementation-contextual-G90.png)
+
+</Tab>
+
+<Tab label="Gray 100">
+
+![Layer set tokens applied to dropdown in the Gray 100 theme](images/color-implementation-contextual-G100.png)
+
+</Tab>
+
+</Tabs>
+
+</Column>
+</Row>
+
+<br />
+
+#### Example of how to apply contextual layers tokens in a layout
+
+Referencing the image below, the starting base level is still the page area
+behind and above the tabs; it uses `$background`. However, there are no sets so
+with each level added simply reuse the same tokens. The tabs and the tile live
+on different levels but they both use the `$layer` token that will output a
+different values in code with the help of the layer component.
+
+<Row>
+<Column colLg={12}>
+
+<Tabs>
+
+<Tab label="White">
+
+![In context contextual token example in the White theme](images/color-implementation-contextual-2-W.png)
+
+</Tab>
+
+<Tab label="Gray 10">
+
+![In context contextual token example in the Gray 10 theme](images/color-implementation-contextual-2-G10.png)
+
+</Tab>
+
+<Tab label="Gray 90">
+
+![In context contextual token example in the Gray 90 theme](images/color-implementation-contextual-2-G90.png)
+
+</Tab>
+
+<Tab label="Gray 100">
+
+![In context contextual token example in the Gray 100 theme](images/color-implementation-contextual-2-G100.png)
+
+</Tab>
+
+</Tabs>
+
+</Column>
+</Row>
+
+#### Building components with contextual tokens
+
+Build a single component as you would normally but apply the contextual color
+tokens instead of the layer set tokens. Then in your product code use the layer
+component to express the nested layer visuals. Even if the component is never
+used at other layer levels it is still acceptable and encouraged to use the
+contextual tokens.
+
+Note: Contextual tokens are only available in code and are not a part of the
+design assets. Designer should use the layer set tokens when creating assets but
+can include contextual tokens in their specs and redlines.
+
+<Row>
+<Column colLg={8}>
+
+![Component spec example using contextual tokens](images/color-implementation-contextual-3.png)
+
+</Column>
+</Row>
+
+### Using the layer component
+
+The layer component is used with the contextual tokens to implement the layering
+model in code. The layer component and contextual tokens are essentially a short
+cut for development because the layer component will apply the correct values to
+a single component without having to develop explicit hard coded variants.
+
+<InlineNotification>
+
+**Coming soon:** Developer documentation is currently be worked on. Check back
```

Comment:
Some Layer dev docs that we have up: https://carbon-react-next.netlify.app/?path=/docs/components-layer--default

Reply:
@joshblack Do we ever want to have those docs in the storybook on the website? Or should we just link out to them and not has a specific section for dev docs on this page?

Reply:
I'm down for either, whichever makes more sense! Just wanted to share what we had in storybook.

Reply:
OK i think we'll just link out to storybook for now since its easy and if we want to incorporate dev docs more intermittently like we do on the grid page we can update it later.